.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
```

Comment:
The folder `.vscode` should not be pushed. Add it to the `.gitignore` and delete this folder